components/NavBar.js

```diff
@@ -17,6 +16,7 @@ export default function NavBar (props) {
         <ul>
           <li>
             <Link href='/'>
+              {/* eslint-disable jsx-a11y/anchor-is-valid */}
```

Comment:
I'm not sure how links work with next, but could this be a button maybe to avoid the linting error?

Reply:
Just refactored this a bit to use buttons.

Reply:
Because this is the navigation bar, these should stay as `<a>`. Or, or does `<Link>` already render as `<a>`?